packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -1380,60 +1381,64 @@ export const addComment: AsyncAction<{
   username: string;
 }> = async ({ state, effects }, { sandboxId, comment, username }) => {
   const id = `${comment}-${username}`;
-  const optimisticComment = {
-    id,
-    insertedAt: new Date().toString(),
-    isResolved: false,
-    originalMessage: {
+  if (state.user) {
```

Comment:
Why don't you just return if `!state.user`?
This makes the rest of the code less indented, which makes the code more readable imo.

```ts
if(!state.user) {
  return;
}

const optimisticComment = {
  // ...
};
// ...
```

Reply:
Good point, fixed :)